frontend/packages/core/src/stories/buttongroup.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { action } from "@storybook/addon-actions"; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Got the tslint error `error  '@storybook/addon-actions' should be listed in the project's dependencies. Run 'npm i -S @storybook/addon-actions' to add it  import/no-extraneous-dependencies`. Was unsure why I was getting the lint error for this storybook import and not the others that are in [dev dependencies ](https://github.com/lyft/clutch/blob/4d0b7faab3f2b5cc4c5b6bb9de84eb52b174d393/frontend/package.json#L65).

Came across https://github.com/storybookjs/storybook/issues/1992 and the recommendation was just to silence the ts error. Thoughts?

Reply:
Let's disable this rule in the eslint config for core to prevent others from having to comment if needed later as well.

something like:
```
    {
      files: ["**/*.stories.*"],
      rules: {
        "import/no-extraneous-dependencies": ["off"],
      }
    },

```